internal/rest/avi_obj_vsvip.go

```diff
@@ -523,14 +525,15 @@ func (rest *RestOperations) AviVsVipCacheDel(rest_op *utils.RestOp, vsKey avicac
 	return nil
 }
 
-func networkNamesToVips(vipNetworks []akov1alpha1.AviInfraSettingVipNetwork) []*avimodels.Vip {
+func networkNamesToVips(vipNetworks []akov1alpha1.AviInfraSettingVipNetwork, enablePublicIP bool) []*avimodels.Vip {
 	var vipList []*avimodels.Vip
 	autoAllocate := true
 	for vipIDInt, vipNetwork := range vipNetworks {
 		vipID := strconv.Itoa(vipIDInt + 1)
 		newVip := &avimodels.Vip{
-			VipID:          &vipID,
-			AutoAllocateIP: &autoAllocate,
+			VipID:                  &vipID,
+			AutoAllocateIP:         &autoAllocate,
+			AutoAllocateFloatingIP: &enablePublicIP,
```

Comment:
Is this a network specific field? @aaha97 
Do we see this getting enabled for some networks specifically in multivip scenarios? I think in that case we'll have to have this as follows
in values.yaml
```
vipNetwokrList:
  - networkName: net1
     enablePublicIP: true
```

and in infrasetting 
```
spec:
  network:
    vipNetworks:
    - networkName: net123
       enablePublicIP: true  
```

The new schema just got in (https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/commit/6a13b776a52e458084b2b1696b787e5a91642c3b) so you might want to rebase.

Reply:
i believe we won't be having the enablePublicIP flag as part of values.yaml, only via aviinfrasetting
all networks will have public ip enabled in case of multi vip